app/api.go

```diff
@@ -18,13 +18,18 @@ type BuildPreProcessor interface {
 }
 
 // CreateApp creates an application from the specified json application descriptor
-func CreateApp(env env.Project, appJson string) error {
+func CreateApp(env env.Project, appJson string, appName string) error {
 
 	descriptor, err := ParseAppDescriptor(appJson)
 	if err != nil {
 		return err
 	}
 
+	if appName != "" {
+		appJson = strings.Replace(appJson, `"` + descriptor.Name + `"`, `"` + appName + `"`, 1)
```

Comment:
@fm-tibco This method very dangers, it might replace wrong name. For example:  The sequence of the Json might be changed after format by some softwares(subline or atom).  

I format my flogo.json with sublime editor and actions node go first then description and name.  It might be replace wrong place if there are some strings same with app name exist inside actions node.